wasm/src/decoder/code_section.rs

```diff
@@ -0,0 +1,32 @@
+use super::common::CodeBlock;
+use super::util;
+use crate::error::Result;
+
+pub fn decode_code_section<R: crate::Reader>(reader: &mut R) -> Result<CodeBlock> {
+    let body_size = util::decode_u32(reader)? as u64;
+    let offset = reader.seek(std::io::SeekFrom::Current(0))?;
```

Comment:
Nit: I believe there's an alias to 0, as Start, or something of the sort

Reply:
`SeekFrom::Start` refers to the first byte and not the current one. And I do need the current one

Reply:
Yup, I forgot the API